src/components/Menu/VersionMenu.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Select from 'react-select';
+import { DOCUMENTATION_PATH, LATEST_ABLY_API_VERSION_STRING } from '../../../data/transform/constants';
+import { Warning } from '../Notifications';
+import { routeToPage } from './route-to-page';
+import { dropdownIndicatorStyles } from './ReactSelectStyles/dropdown-indicator-styles';
+import { optionStyles } from './ReactSelectStyles/option-styles';
+import { controlStyles } from './ReactSelectStyles/control-styles';
+import { noIndicatorSeparator } from './ReactSelectCustomComponents/no-indicator-separator';
+import { groupHeadingStyles } from './ReactSelectStyles/group-heading-styles';
+import { menuListStyles } from './ReactSelectStyles/menu-list-styles';
+import { noPaddingValueContainerStyles } from './ReactSelectStyles/Versions/no-padding-value-container-styles';
+
+type PageVersion = {
+  node: {
+    version: string;
+    slug: string;
+  };
+};
+
+export type VersionData = {
```

Comment:
Should be called `VersionMenuProps`. Naming convention is loosely `[ComponentName]Props`